File: 02-frontend/library-app/src/models/CheckoutResponse.ts
class CheckoutResponse{
    bookId?: number;
    title?: string;
    author?: string;
    email?: string;
    reservationDate?: string
    img?: string;
    collectionDate?: string

    constructor(bookId: number , title: string,author: string,img: string, email:string,
        reservationDate: string, collectionDate: string){
        this.bookId = bookId;
        this.title = title;
        this.author = author;
        this.reservationDate = reservationDate;
        this.email = email;
        this.img = img;
        this.collectionDate= collectionDate
    }
}

export default CheckoutResponse;

File: 01-backend/springboot-library/src/main/java/com/project/springbootlobrary/responseModels/CheckoutResponse.java
package com.project.springbootlobrary.responseModels;

import lombok.*;
import lombok.experimental.FieldDefaults;

import javax.persistence.Column;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@FieldDefaults(level = AccessLevel.PRIVATE)
public class CheckoutResponse {
    Long bookId;

    String title;

    String author;

    String img;

    String userEmail;

    String reservationDate;

    String collectionDate;
}
